src/templates/cell-line.tsx

```diff
@@ -1,64 +1,88 @@
+} from "../component-queries/types";
+import { convertFrontmatterToNormalCellLines } from "../component-queries/convert-data";
+import { NormalCellLineInfoCard } from "../components/CellLineInfoCard/NormalCellLineInfoCard";
+import { DefaultButton } from "../components/shared/Buttons";
 
-const { container, section } = require("../style/cell-line.module.css");
+const {
+    container,
+    section,
+    leftCard,
+    returnArrow,
+} = require("../style/cell-line.module.css");
+
+const Arrow = require("../img/arrow.svg");
 
 interface QueryResult {
+    location: Location;
     data: {
         markdownRemark: {
-            frontmatter: {
-                cell_line_id: string;
-                clone_number: number;
-                gene: string[];
-                genetic_modifications?: GeneticModification[];
-                status: string;
-                thumbnail_image: any;
+            id: string;
+            key: string;
+            fields: {
+                slug: string;
             };
+            frontmatter: NormalCellLineFrontmatter;
         };
     };
 }
 
-interface CellLineProps {
-    cellLineId: string;
-    cloneNumber: number;
-    geneticModifications?: GeneticModification[];
-    status: string;
-    thumbnail: any;
+interface CellLineProps extends UnpackedNormalCellLine {
+    href: string;
 }
 
 // eslint-disable-next-line
 export const CellLineTemplate = ({
     cellLineId,
     cloneNumber,
-    geneticModifications,
+    tagLocation,
+    taggedGene,
     status,
-    thumbnail,
+    thumbnailImage,
+    href,
+    orderPlasmid,
+    healthCertificate,
+    certificateOfAnalysis,
+    orderLink,
+    protein,
+    fluorescentTag,
+    alleleCount,
 }: CellLineProps) => {
-    const image = getImage(thumbnail);
+    const image = thumbnailImage ? getImage(thumbnailImage) : undefined;
     return (
         <div className={container}>
-            <div className={section}>
-                <h1>AICS-{cellLineId}</h1>
-                <p>Clone Number: {cloneNumber}</p>
-                <p>
-                    Gene:{" "}
-                    {geneticModifications
-                        ?.map((mod) => mod.gene?.frontmatter?.symbol)
-                        .join(" / ")}
-                </p>
-                <p>
-                    Tag:{" "}
-                    {geneticModifications
-                        ?.map((mod) => mod.tag_location)
-                        .join(" / ")}
-                </p>
-                <p>Status: {status}</p>
+            <div className={leftCard}>
+                <Link to="/normal-catalog">
+                    <DefaultButton>
+                        <Arrow className={returnArrow} />
+                        Return to Cell Catalog
+                    </DefaultButton>
+                </Link>
+                <NormalCellLineInfoCard
+                    href={href}
+                    cellLineId={cellLineId}
+                    geneName={taggedGene[0].name}
```

Comment:
why only the first one? this should be the same as the table, where if they have more than one gene they are listed out

Reply:
I'm not saying render a table, I'm saying render it like we do on the table, 
<img width="1770" height="216" alt="Screenshot 2025-07-22 at 11 34 16 AM" src="https://github.com/user-attachments/assets/27f213b2-82c1-4b91-a6aa-b8a0c16799ce" />

Reply:
I think the TS code should be written in a way that we won't have to change it, ie, we know we're going to render all of them. then in the future you should just be able to adjust the css